packages/fxa-content-server/app/scripts/views/sign_in_password.js

```diff
@@ -75,6 +75,11 @@ const SignInPasswordView = FormView.extend({
     }
   },
 
+  logView() {
```

Comment:
For your consideration, I kinda liked how [flow-events-mixi](https://github.com/mozilla/fxa/blob/6453eede031cc238526c83a2e3737a3d7cf0e27f/packages/fxa-content-server/app/scripts/views/mixins/flow-events-mixin.js#L11) worked. You could import mixin and it would automatically hook into backbone lifecycle events/submit. Not saying we replicate that exactly, but something that would allow us to easily instrument a view.

Reply:
With that approach I suppose we could use a view's `viewName` to determine which glean metrics function to call.  Although this view does not explicitly set its viewName and it's a bit difficult to know all the effects of setting one now.  For form submission failures I need to capture the reason for the error as an event property, so we'd need to come up with the wiring for that.  I don't think the complexity is worthwhile at that point.

Thanks for teh suggestion. 👍